content/docs/user-guide/scripting/script-canvas/editor-reference/functions.md

```diff
@@ -70,38 +71,35 @@ The final function graph should look like this:
 
 ![After completing the steps in this example, you should end up with a linear interpolation function.](/images/user-guide/scripting/script-canvas/function-linear-interpolation.png)
 
-1. Start a new function using **File**, **New Function** or by using the function create button in the upper right corner of the canvas.
-
-1. Create the following four variables in **Variable Manager**:
-   + **Start**
-   + **End**
-   + **Time**
-   + **Result**
+1. Start a new function using **File**, **New Script** or by using the function create button in the upper right corner of the canvas.
 
-1. Set the **scope** of *Start*, *End*, and *Time* to **In**. These are now your input parameters.
+1. Click the left side **Create execution nodeling** button to make an input node on the canvas.
 
-1. Set the **scope** of *Result* to **Out**. This is now your return value.
-
-1. Place the **Subtract**, **Multiply**, and **Add** nodes onto your graph and connect them as shown in the previous image.
+1. Click the **+** button next to Add data input to add an input parameter to the execution nodeling. This will prompt you for a name and type of the new variable. Enter **Start** as the name and **Number** as the type. Repeat this twice more for **Time** and **End**
```

Comment:
We should keep the step that instructs readers to add the 3 nodes.
```suggestion
1. Click the **+** button next to Add data input to add an input parameter to the execution nodeling. This will prompt you for a name and type of the new variable. Enter **Start** as the name and **Number** as the type. Repeat this twice more for **Time** and **End**.

1. Place the **Subtract**, **Multiply**, and **Add** nodes onto your graph and connect them as shown in the previous image.
```